packages/flex-plugin-scripts/src/scripts/deploy.ts

```diff
@@ -58,56 +60,56 @@ export const _verifyPath = (baseUrl: string, build: Build): boolean => {
   return existingAssets.every(checkPathIsUnused) && existingFunctions.every(checkPathIsUnused);
 };
 
+/**
+ * Returns the default version of flex-ui dependency if uiDependency is empty
+ * @param uiVersion the Flex UI version
+ * @param uiDependencies   the provided ui dependency
+ */
+export const _getDefaultUIDependencies = (uiVersion: string, uiDependencies: UIDependencies): UIDependencies => {
+  if (!uiDependencies.react || !uiDependencies['react-dom']) {
+    if (versionSatisfiesRange(uiVersion, '>=2.0.0')) {
```

Comment:
shouldn't this be `>=2.0.0-alpha.0`?

Reply:
https://github.com/twilio/flex-plugin-builder/pull/559/files#diff-29dab0975603dcd7a9790e3b52528cc995ab150dcdc6d7f113876bfc3a33fa8bR14

Reply:
ahh 👌